packages/@coorpacademy-components/src/molecule/resource-player/index.js

```diff
@@ -135,7 +136,11 @@ class ResourcePlayer extends React.Component {
     return (
       <div data-name={type} className={className}>
         {overlayView}
-        <ResourceElement {...this.props} autoplay={this.state.autoplay} />
+        <ResourceElement
+          {...this.props}
+          autostart={!this.props.overlay}
+          autoplay={this.state.autoplay}
+        />
```

Comment:
Missing prop type definition in this file

Reply:
added within `ResourceElement.propTypes`